lib/messaging/messagehandler_message.rb

```diff
@@ -10,4 +10,9 @@ def initialize
     handled = false
   end
 
+  def handle_message(message)
+    message.handled = true
+    puts 'This is probably where the message gets handled.'
```

Comment:
The messages are handled in the objects derived from messagehandler_base.rb You have to add a white list that says to the object what type of messages it can handle. For each type of message you add a function. The base object will then call that function.